packages/plugins/rubygem-base64/rubygem-base64.spec

```diff
@@ -0,0 +1,61 @@
+# template: default
+%global gem_name base64
+
+Name: rubygem-%{gem_name}
+Version: 0.2.0
+Release: 1%{?dist}
+Summary: Support for encoding and decoding binary data using a Base64 representation
+License: Ruby and BSD-2-Clause
+URL: https://github.com/ruby/base64
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: ruby >= 2.4
+BuildRequires: ruby >= 2.4
+BuildRequires: rubygems-devel
+BuildArch: noarch
+# end specfile generated dependencies
+
+%description
+Support for encoding and decoding binary data using a Base64 representation.
+
+
+%package doc
+Summary: Documentation for %{name}
+Requires: %{name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+%setup -q -n  %{gem_name}-%{version}
+
+%build
+# Create the gem as gem install only works on a gem file
+gem build ../%{gem_name}-%{version}.gemspec
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%gem_install
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%license %{gem_instdir}/LICENSE.txt
+%{gem_libdir}
+%exclude %{gem_cache}
+%{gem_spec}
+
+%files doc
+%doc %{gem_docdir}
+%doc %{gem_instdir}/README.md
+
+%changelog
+* Fri Aug 23 2024  0.2.0-1
```

Comment:
changelog is missing name and optionally email

Reply:
Added my name